### ISSUE TITLE
Compile Grammar causes `this.tokenizedBuffer.reloadGrammar is not a function`

### ISSUE THREAD

**Issue by bentolor** [asciidoctor/atom-language-asciidoc#163]
## Description

I'm trying to add Kotlin to the supported language bindings. Therefore I installed the language-kotlin package and added  this to `languages.cson`:

```
,
  pattern: 'kotlin|kts?', type: 'source', code: 'Kotlin'
```

I also manually hacked the `language-asciidoc.cson` (w/o success) until I realized it should become generated. Now, when I try to perform _Asciidoc Grammar: Compile Grammar and Reload_ I only see this exception:

```
/home/andrei/atom-1.11.1/out/app/src/text-editor.coffee:2816 
Uncaught TypeError: this.tokenizedBuffer.reloadGrammar is not a function
module.exports.CommandPaletteView.confirmed @ /home/andrei/atom-1.11.1/out/app/node_modules/command-palette/lib/command-palette-view.coffee:183
```

Strange home path btw. Any ideas?

## Disclaimer
I actually have no real clue what I'm actually doing…

**Comment by ldez** [comment]
Hello,

Are you running Atom in Dev Mode ?

**Comment by bentolor** [comment]
@ldez Yes. Otherwise [this safeguard would take effect](https://github.com/asciidoctor/atom-language-asciidoc/blob/master/lib/main.coffee#L51)

**Comment by bentolor** [comment]
What I did so far:

1. I'm using Atom 1.11.2 on Ubunut
2. I've installed `asciidoc-assistant`
3. I've uninstalled `language-asciidoc`, exited Atom
4. I checked out this repository, run `apm install` and then `apm link --dev`
5. Started `atom . --dev` and trying to run the command above

**Comment by ldez** [comment]
I know, I writing the safeguard ;)

**Comment by ldez** [comment]
You don't need to uninstall `language-asciidoc` in normal mode because the package is automatically replace by the dev version (if you have install it before) when you are in Dev Mode.

**Comment by ldez** [comment]
I reproduce with the last version of Atom.

**Comment by ldez** [comment]
It's an issue with Atom: https://github.com/atom/atom/issues/13022

**Comment by ldez** [comment]
Fixed in the next version

**Comment by bentolor** [comment]
Thanks, @ldez  for your fast & helpful response!

**Comment by bradennapier** [comment]
im still having this issue in 1.18 ?

**Comment by ldez** [comment]
@bradennapier could you create a new issue?

**Comment by ldez** [comment]
@bradennapier please update the plugin (atom-language-asciidoc)

**Comment by bradennapier** [comment]
i dont see that available anywhere even if i try to install it

**Comment by ldez** [comment]
I don't reproduce with Atom 1.18 and language-asciidoc 1.10.

Please uninstall `language-asciidoc` and re-install.

Open a new issue if you still have a problem.
